src/Symfony/Component/Validator/Constraints/Traverse.php

```diff
@@ -27,12 +28,17 @@ class Traverse extends Constraint
     /**
      * @param bool|array<string,mixed>|null $traverse Whether to traverse the given object or not (defaults to true). Pass an associative array to configure the constraint's options (e.g. payload).
      */
+    #[HasNamedArguments]
     public function __construct(bool|array|null $traverse = null)
```

Comment:
this constraint seems to be missing group and payload as arguments to be able to configure them without options

Reply:
fixed